packages/web/src/scss/tools/_form-fields.scss

```diff
@@ -162,3 +176,11 @@
 @mixin item-label-checked() {
     @include typography.generate(form-fields-theme.$item-label-checked-typography);
 }
+
+@mixin helper-text() {
+    @include typography.generate(form-fields-theme.$item-message-typography);
```

Comment:
should have its own or common (renamed) variables. Please do not mix it with the message (in the name at least).

Reply:
fixed